Pod/Classes/SEGAmplitudeIntegration.m

```diff
@@ -10,15 +10,18 @@ - (id)initWithSettings:(NSDictionary *)settings
     if (self = [super init]) {
         self.settings = settings;
```

Comment:
Not related to this change, but the pattern is used incorrectly. You shouldn't be assigning the fields `self` here (the other initializer does that already).

Something like 
```
- (id)initWithSettings:(NSDictionary *)settings
{
  return [self initWithSettings:settings andAmplitude:[Amplitude instance] andAmpRevenue:[AMPRevenue revenue]];
}
```

Reply:
https://github.com/segment-integrations/analytics-ios-integration-amplitude/pull/32